command/pki_issue_intermediate.go

```diff
@@ -0,0 +1,351 @@
+		c.UI.Error(fmt.Sprintf("Error Signing Intermiate On %v", err))
+		return 1
+	}
+	// Success!  Save Our Progress (and Parse the Response)
+	failureState.csrSigned = true
+	serialNumber := rootResp.Data["serial_number"].(string)
+	failureState.certSerialNumber = serialNumber
+
+	caChain := rootResp.Data["ca_chain"].([]interface{})
+	caChainPemBundle := ""
+	for _, cert := range caChain {
+		caChainPemBundle += cert.(string) + "\n"
+	}
+	failureState.caChain = caChainPemBundle
+
+	// Next Import Certificate
+	certificate := rootResp.Data["certificate"].(string)
+	issuerId, err := importIssuerWithName(client, intermediateMount, certificate, c.flagNewIssuerName)
+	failureState.certIssuerId = issuerId
+	if err != nil {
+		if strings.Contains(err.Error(), "error naming issuer") {
+			failureState.certImported = true
+			c.UI.Error(failureState.generateFailureMessage())
+			c.UI.Error(fmt.Sprintf("Error Naming Newly Imported Issuer: %v", err))
+			return 1
+		} else {
+			c.UI.Error(failureState.generateFailureMessage())
+			c.UI.Error(fmt.Sprintf("Error Importing Into %v Newly Created Issuer %v: %v", intermediateMount, certificate, err))
+			return 1
+		}
+	}
+	failureState.certImported = true
+
+	// Then Import Issuing Certificate
+	issuingCa := rootResp.Data["issuing_ca"].(string)
+	_, err = importIssuerWithName(client, intermediateMount, issuingCa, parentIssuerName)
+	if err != nil {
+		if strings.Contains(err.Error(), "error naming issuer") {
+			c.UI.Warn(fmt.Sprintf("Unable to Set Name on Parent Cert from %v Imported Into %v with serial %v, err: %v", parentIssuerName, intermediateMount, serialNumber, err))
+		} else {
+			c.UI.Error(failureState.generateFailureMessage())
+			c.UI.Error(fmt.Sprintf("Error Importing Into %v Newly Created Issuer %v: %v", intermediateMount, certificate, err))
+			return 1
+		}
+	}
+
+	// Finally Import CA_Chain (just in case there's more information)
+	if len(caChain) > 2 { // We've already imported parent cert and newly issued cert above
+		importData := map[string]interface{}{
+			"pem_bundle": caChainPemBundle,
+		}
+		_, err := client.Logical().Write(intermediateMount+"/issuers/import/cert", importData)
+		if err != nil {
+			c.UI.Error(failureState.generateFailureMessage())
+			c.UI.Error(fmt.Sprintf("Error Importing CaChain into %v: %v", intermediateMount, err))
+			return 1
+		}
+	}
+	failureState.caChainImported = true
+
+	// Finally we read our newly issued certificate in order to tell our caller about it
+	c.readAndOutputNewCertificate(client, intermediateMount, issuerId)
+
+	return 0
+}
+
+func (c *PKIIssueCACommand) readAndOutputNewCertificate(client *api.Client, intermediateMount string, issuerId string) {
+	resp, err := client.Logical().Read(sanitizePath(intermediateMount + "/issuer/" + issuerId))
+	if err != nil || resp == nil {
+		c.UI.Error(fmt.Sprintf("Error Reading Fully Imported Certificate from %v : %v",
+			intermediateMount+"issuer/"+issuerId, err))
```

Comment:
nit: error message is missing a '/' between intermediateMount and issuer

Reply:
Ack, fixed